submissions/evgeniy24/html-css-popup-demo/index.html

```diff
@@ -0,0 +1,229 @@
+    <link
+      rel="apple-touch-icon"
+      sizes="57x57"
+      href="./favicons/apple-icon-57x57.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="60x60"
+      href="./favicons/apple-icon-60x60.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="72x72"
+      href="./favicons/apple-icon-72x72.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="76x76"
+      href="./favicons/apple-icon-76x76.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="114x114"
+      href="./favicons/apple-icon-114x114.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="120x120"
+      href="./favicons/apple-icon-120x120.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="144x144"
+      href="./favicons/apple-icon-144x144.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="152x152"
+      href="./favicons/apple-icon-152x152.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="./favicons/apple-icon-180x180.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="./favicons/android-icon-192x192.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./favicons/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="96x96"
+      href="./favicons/favicon-96x96.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="./favicons/favicon-16x16.png"
+    />
+    <!--    <link rel="manifest" href="./favicons/manifest.json">-->
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" />
+    <meta name="theme-color" content="#ffffff" />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="list__item">
+            <a class="list__item_link" href="#">Omail</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item_link" href="#">Images</a>
+          </li>
+          <li class="list__item">
+            <input id="popup-menu" class="check__input" type="checkbox" />
+            <label for="popup-menu" class="label__input">
+              <span class="list__item_link item__apps"></span>
+            </label>
+            <div class="popup">
+              <ul class="popup__menu">
+                <li class="popup__item">
+                  <a href="#" class="item__link">
+                    <figure class="item__icon icon__docs icon"></figure>
+                  </a>
+                  <span class="item__text">Docs</span>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="item__link">
+                    <figure class="item__icon icon__translate icon"></figure>
```

Comment:
HTML element `<figure>` better use with `<img>` tag as child element it's more semantically and give you a small benefit in scanning by search bot. [Read more about figure element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)
If you want to use icons as background images better use a different tag such as `<div>` or `<span>` for it.